docs/developer/release/release.md

```diff
@@ -0,0 +1,59 @@
+# Release
+
+Once we are confident that the release will be out shortly, and doesn't need any more changes:
+
+- [ ] Create a PGP-signed git tag for the version, e.g., for dangerzone `v0.1.0`:
+
+  ```bash
+  git tag -s v0.1.0
+  git push origin v0.1.0
+  ```
+
+- [ ] Create an archive of the Dangerzone source in `tar.gz` format:
+
+  ```bash
+  export VERSION=$(cat share/version.txt)
+  git archive --format=tar.gz -o dangerzone-${VERSION:?}.tar.gz --prefix=dangerzone/ v${VERSION:?}
+  ```
+
+  _(This is covered by our automated build steps)_
+
+- [ ] Run container scan on the produced container images (some time may have passed since the artifacts were built)
+
+  ```bash
+  docker pull anchore/grype:latest
+  docker run --rm -v ./share/container.tar:/container.tar anchore/grype:latest /container.tar
+  ```
+
+- [ ] Collect the assets in a single directory, calculate their SHA-256 hashes, and sign them.
+  There is an `./dev_scripts/sign-assets.py` script to automate this task.
+
+  ```bash
+  # Sign all the assets
+  ./dev_scripts/sign-assets.py ~/release-assets/$VERSION/github --version $VERSION
+  ```
+
+- [ ] Upload all the assets to the draft release on GitHub.
+
+  ```bash
+  find ~/release-assets/$VERSION/github | xargs -n1 ./dev_scripts/upload-asset.py --token ~/token --draft
+  ```
+
+- [ ] Update the draft release to target the final git tag.
+
+- [ ] Send a PR to update the [Dangerzone website](https://github.com/freedomofpress/dangerzone.rocks) to link to the new installers.
+
+- [ ] Send a PR that updates the Dangerzone version and links to our installation instructions (`INSTALL.md`) in `README.md`
+
+## 📣 Publish the release!
+
+Up until this point, we can take back any artifact we have created, because
+nothing is published. To actually publish the release, we can do the following:
+
+- [ ] Merge the PRs in the `apt-tools-prod` and `yum-tools-prod` repos.
+- [ ] Make the GitHub draft release public.
+- [ ] Merge the PRs in `freedomofpress/dangerzone.rocks` and `freedomofpress/dangerzone`.
+- [ ] Toot release announcement on our mastodon account https://social.freedom.press/@dangerzone
+- [ ] Update the brew cask release of Dangerzone with a [PR like this one](https://github.com/Homebrew/homebrew-cask/pull/116319)
```

Comment:
I'm actually not sure we should keep this here, since we're currently not doing it and brew isn't a supported installation way. 🤔

Reply:
I was thinking about this as well. Also, for the past releases, the Dangerzone cask was always prepared by other Homebrew users, so I guess we can say it's off our hands. Changed in 53393de170ecfb3d2aeaa39cc6ee8fe83cecfb4b.